Dockerfile

```diff
@@ -0,0 +1,133 @@
+#####################################
+FROM golang:1.18.1-buster AS lotus-builder
```

Comment:
Probably want a newer Go version

Reply:
1.18.1 is what we currently build all our builds with (see https://github.com/filecoin-project/lotus/blob/master/.circleci/config.yml#L9 and https://github.com/filecoin-project/lotus/blob/master/GO_VERSION_MIN).

 If we want a newer version, we should update that across the board, and probably in a separate PR.

Reply:
Discussed in standup: @ianconsolata can you please update to 1.18.8, and enforce it in the Makefile (in a separate PR)?

Reply:
Once that one is merged, I will update this PR to include the changes.